### PR TITLE
Add rollbar env vars to README and .env.example.

### PR DESCRIPTION
**Pull request by squaresurf**
Why:

* All custom env vars should be documented.

This change addresses the need by:

* Add rollbar env vars to README and .env.example.
* Sort env vars alphabetically in the README and the .env.example.

### CODE REVIEW

**Comment by Mrjaco12** [comment]
:+1:

File: README.md [hunk 1]
```diff
@@ -21,16 +21,16 @@ The current ruby development version is 2.3.0.
...
-
 ### `APPLICATION_ENVIRONMENT`
 
 When you need to write code to sense if the environment is one of (development,
 test, staging, or production), then you should use this env var instead of
 `RACK_ENV` or `RAILS_ENV`.
 
+### `APPLICATION_HOST`
+
+If set, this will setup rack canonical host to redirect to it no matter what was requested.
```
Comment (samcdavid) [inline README.md]: That is only for the root domain right? The wording here is a little concerning, but it is also late and I should probably just go to bed :smile:

Comment (squaresurf) [inline README.md]: Good catch. I actually didn't write this today, but just moved it so it was alphabetical. I'll fix it up :)

**Comment by jasontwong** [comment]
:+1:

**Comment by squaresurf** [comment]
@samcdavid would you take another gander at the README?

**Comment by samcdavid** [comment]
:+1:

**Pull request merged**